src/components/cafes/[id]/FlavorItem/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { dot, flavorItem, flavorList } from './FlavorItem.css';
+
+export interface FlavorListProps {
+  fruity: string;
+  floral: string;
+  sweet: string;
+  spices: string;
+  roasted: string;
+  green: string;
+  sour: string;
+  nutty: string;
+  other: string;
+}
+
+export const flavorNames: FlavorListProps = {
+  fruity: '과일',
+  floral: '꽃',
+  sweet: '달콤한',
+  spices: '향신료',
+  roasted: '구운',
+  green: '그린',
+  sour: '시큼',
+  nutty: '견과',
+  other: '기타',
+};
```

Comment:
아래처럼 바꾸면 간결해질 것 같은데 어떤가요?

```suggestion
export const FLAVOR_NAME = {
  fruity: '과일',
  floral: '꽃',
  sweet: '달콤한',
  spices: '향신료',
  roasted: '구운',
  green: '그린',
  sour: '시큼',
  nutty: '견과',
  other: '기타',
} as const;

export type Flavor = keyof typeof FLAVOR_NAME
```